mobile_verifier/migrations/13_modeled_coverage.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TYPE signal_level as ENUM (
+       'none',
+       'low',
+       'medium',
+       'high'
+);
+
+CREATE TABLE hex_coverage (
+       uuid UUID NOT NULL,
+       hex BIGINT NOT NULL,
+       indoor BOOLEAN NOT NULL,
+       cbsd_id TEXT NOT NULL,
+       signal_level signal_level NOT NULL,
+       coverage_claim_time TIMESTAMPTZ NOT NULL
```

Comment:
need a `,` at the end of this line and I think this migration needs to be renamed to `15_modeled_coverage.sql`